React/Libraries.md

```diff
@@ -0,0 +1,87 @@
+
+##### Next.js Routing
+- [next-connect](https://github.com/hoangvvo/next-connect) - Minimal router and middleware layer for Next.js
+
+##### React Styling and Component Libraries
+- [styled-components](https://github.com/styled-components/styled-components) - Visual primitives for the component age
+- [emotion](https://github.com/emotion-js/emotion) - CSS-in-JS library designed for high performance style composition
+- [chakra-ui](https://github.com/chakra-ui/chakra-ui/) - Simple, Modular & Accessible UI Components for your React Applications
+- [styled-system](https://github.com/styled-system/styled-system) - Style props for rapid UI development
+
+##### React Components
+- [react-select](https://github.com/JedWatson/react-select) - The Select Component for React.js
+- [react-datepicker](https://github.com/Hacker0x01/react-datepicker/) - A simple and reusable datepicker component for React
+- [react-virtualized](https://github.com/bvaughn/react-virtualized) - React components for efficiently rendering large lists and tabular data
+- [react-virtuoso](https://github.com/petyosi/react-virtuoso) - An elegant virtual list component for React
+- [@tippyjs/react](https://github.com/atomiks/tippyjs-react) - The complete tooltip, popover, dropdown, and menu solution for the web, powered by Popper
+- [downshift](https://github.com/downshift-js/downshift) - A set of primitives to build autocomplete, combobox or select dropdown React components
+- [react-collapse](https://github.com/nkbt/react-collapse) - Component-wrapper for collapse animation with react-motion for elements with variable (and dynamic) height
+- [react-markdown](https://github.com/remarkjs/react-markdown) - Markdown component for React
+- [react-slider](https://github.com/zillow/react-slider) - Accessible, CSS agnostic, slider component for React
+- [react-qr-code](https://github.com/rosskhanas/react-qr-code) - A QR code generator for React and React Native
+- [react-modal](https://github.com/reactjs/react-modal) - Accessible modal dialog component for React
+- [react-cool-inview](https://github.com/wellyshen/react-cool-inview) - React hook to monitor an element enters or leaves the viewport (or another element)
+- [react-avatar-editor](https://github.com/mosch/react-avatar-editor) - Small avatar & profile picture component that allows resizing and cropping uploaded images using a intuitive user interface
+- [react-rating](https://github.com/dreyescat/react-rating) - A rating react component with custom symbols
+- [react-compound-slider](https://github.com/sghall/react-compound-slider) - A small React slider with no opinion on markup or styles
+- [react-slick](https://github.com/akiran/react-slick) - React carousel component
+- [@datepicker-react/hooks](https://github.com/tresko/react-datepicker/tree/master/packages/hooks) - An easily internationalizable, accessible, mobile-friendly datepicker library for the web, build with styled-components
+
```

Comment:
this one also https://kyleamathews.github.io/react-headroom/

Reply:
Added